media/stylus/components/compat-tables/bc-history.styl

```diff
@@ -0,0 +1,112 @@
+/*
+History
+====================================================================== */
+
+.bc-history {
+    z-index: 20;
+    box-sizing: border-box;
+    border-top: 4px solid $bc-color-border;
+    border-bottom: 3px solid $bc-color-border;
+    overflow: hidden;
+    cursor: default;
+}
+
+    .bc-table .bc-history{
+        position: absolute;
+        top: 100%;
+        display: none;
+        visibility: hidden;
+        box-shadow: inset 200px 0 0 0 $bc-color-border;
+    }
+
+    .active .bc-history {
+        display: block;
+        visibility: visible;
+        height: 0;
+        padding-left: 200px;
+        transition: height 0.2s ease-out;
+    }
+
+    .bc-history:before {
+        content: '';
+        display: block;
+        height: 45px;
+        width: 60px;
+        float: right;
+    }
```

Comment:
![To the left, to the left](https://cloud.githubusercontent.com/assets/933396/8113453/207f4146-103c-11e5-86c9-17bf73b0265d.gif)